salt/modules/linux_lvm.py

```diff
@@ -60,10 +60,18 @@ def fullversion():
     return ret
 
 
-def pvdisplay(pvname=''):
+def pvdisplay(pvname='', real=False):
     '''
     Return information about the physical volume(s)
 
+    pvname
+        physical device name
+    real
+        dereference any symlinks and report the real device
+
+        .. versionadded:: 2015.8.5
+
+
     CLI Examples:
```

Comment:
Hah! Wasn't out when I fetched upstream this (my) morning. Thanks for catching that.